.docker.env

```diff
@@ -0,0 +1,11 @@
+# NOTE TO DEVELOPERS
+# Please do not directly change the values of the variables specified in this file, as this may lead to unexpected behavior of your local development server.
+## Additionally, changing the variables (which are flags for running the local development server) to true will permanently enable the flag to your `make run-devserver` command even you do not explicitly pass the flag.
```

Comment:
```suggestion
# Changing the variables (which are flags for running the local development server) to true will enable the flag to your `make run-devserver` command even if you do not explicitly pass the flag.
```

Does this more clearly communicate what you want?

Reply:
done